src/app/modules/gb-data-selection-module/constraint-components/gb-combination-constraint/gb-combination-constraint.component.ts

```diff
@@ -54,19 +55,8 @@ export class GbCombinationConstraintComponent extends GbConstraintComponent impl
   }
 
   onDropdown(event) {
-    let results = this.constraintService.searchAllConstraints('');
-
-    // Workaround for dropdown not showing properly, as described in
-    // https://github.com/primefaces/primeng/issues/745
-    this.searchResults = [];
-    this.searchResults = results;
-    event.originalEvent.preventDefault();
-    event.originalEvent.stopPropagation();
-    if (this.autoComplete.panelVisible) {
-      this.autoComplete.hide();
-    } else {
-      this.autoComplete.show();
-    }
+    this.searchResults = this.constraintService.searchAllConstraints('');
+    UIHelper.removePrimeNgLoaderIcon(this.element, 200);
```

Comment:
is this issue still there in the newest version?

Reply:
apparently still there, but takes on a different form. Clicking the dropdown button will only show the dropdown but not close it, the user needs to click somewhere on the screen to close it. A tolerable behavior I would say

Reply:
ah ok, I was just curious how fast they are with fixing issues